scripts/apps/archive/directives/MediaPreview.ts

```diff
@@ -90,6 +92,28 @@ export function MediaPreview(api, $rootScope, desks, superdesk, content, storage
                 return _.map(scope.item.company_codes, 'qcode').join(', ');
             };
 
+            /**
+             * @ngdoc method
+             * @name associationExists
+             * @private
```

Comment:
> If the comment only repeats what is in the function signature, it doesn't provide any additional information and isn't worth having.

I meant to suggest to you to simply remove the comment. If the function signature is simple and clear, as it is in this case, no comments are needed. Can you remove all such comments in this PR?

Reply:
Remove from all those files that are included in this PR or from other files too?

Reply:
Only those in this PR.